packages/quantic/force-app/main/default/lwc/quanticRefineToggle/quanticRefineToggle.html

```diff
@@ -27,6 +27,11 @@
     </div>
     <div class="slds-align_absolute-center" slot="content">
       <div class="refine-modal__content">
+        <template if:true={refineButtonDisabled}>
+          <div class="slds-align_absolute-center slds-text-color_weak refine-modal__empty-message">
+            {labels.noFiltersAvailableForThisQuerInTheCurrentTab}
```

Comment:
There is a typo in the label key.

```suggestion
            {labels.noFiltersAvailableForThisQueryInTheCurrentTab}
```